version.json

```
@@ -1,3 +1,3 @@
 {
-  "version": "v0.8.1"
+  "version": "v0.8.2"
```

```suggestion
  "version": "v0.9.0"
```

#194 is breaking for our main library call so we probably ought to signal that.